website/content/docs/secure/vault/index.mdx

```diff
@@ -54,12 +54,14 @@ Jobs that need access to Vault may specify which Vault cluster to use with the
 
 ## Compatibility
 
-* Nomad versions 1.4.0 and above are compatible with any currently supported
-  version of Vault.
-
-|              | Vault 1.13.0+ |
-|--------------|---------------|
-| Nomad 1.4.0+ | ✅            |
```

Comment:
😅 

As much as I'd hate to add more manual steps to our release checklist, I wonder if we can make this more automatic.